roles/validations/README.md

```diff
@@ -0,0 +1,14 @@
+# validations
+The QE Validations role provides a framework for performing state verification.
+This is useful for QE purposes and will be leveraged post-deployment to assert
+that required conditions have been satisfied. For example, if I update a certain
+value in my `OpenStackDataPlaneNodeSet`. I want to validate that the change has
+been applied as per my expectations on the relevant OpenStack Compute nodes.
+
+## Privilege escalation
+If apply, please explain the privilege escalation done in this role.
+
+## Parameters
+* `param_1`: this is an example
```

Comment:
this will also probably depend on the validations to run.

You can add the top-level ones though:
```
cifmw_validations_list: []
cifmw_validations_run_all: false
cifmw_validation_default_path: "{{ role_path }}/tasks"
```

and add a mention about "Check validations set you want to run for more parameters".

Unless we enforce README update when adding new validation - but it may end with an endless file. Not really sure about the best way to handle this. Maybe add a comment section at the top of each validation file?

In any cases, once the rule is defined, it must be shown in this README here.